### DISCUSSION TITLE
Connecting to multiple Azure environments using different credentials

### DISCUSSION THREAD

**Discussion by byunk24** [Azure/CCOInsights#294] [Q&A] [answered]
Is it possible to connect to multiple environments using different credentials?  We have our prod/qa environment under different credentials than our dev/test environment.  TIA!

**Comment by josunefon** [comment, marked as answer]
Hi @byunk24 currently this is not supported, you need to use the same user/credentials. However, you can alway summit a feature request from [here](https://github.com/Azure/CCOInsights/issues/new?assignees=&labels=&template=feature_request.md)
